### PR TITLE
pr134

### PR DESCRIPTION
**Pull request by zzzeek**
change some things.

### CODE REVIEW

**Comment by zzzeek** [comment]
plain comment

**Comment by zzzeek** [comment]
plain comment

**Comment by zzzeek** [comment]
another plain comment

File: bar.py [hunk 1]
```diff
@@ -13,14 +13,14 @@ def bar_one(x, y):
...
 def bar_two(z, q):
     bat.do_z(z)
-    if name:
-        bat.do_q(q)
-    else:
-        bat.do_q(q + 7)
+    bat.do_q(q + name)
 
 
 def bar_three(a, g, h):
```
Comment (zzzeek) [inline bar.py]: so here we are reviewing on the change that is not the current gerrit revision.  WILL IT WORK???

**Comment by sqla-tester** [comment]
Patchset c9ff63e4be384668ad798fd8cc0c359ffd3f0424 added to existing Gerrit review https://gerrit.sqlalchemy.org/#/c/sqlalchemy/testgerrit/+/1046

File: foo.py [hunk 2]
```diff
@@ -3,26 +3,27 @@
...
     bar.do_y(y)
+    bat.do_z(z)
 
 
 def foo_two(z, x, y):
     """do foo two"""
     bat.do_z(z * 12)
     bar.do_x(x).y(y).z(z)
-    bar.do_y(y)
+    bar.do_y(y + 9)
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

so I see you did the thing

File: foo.py [hunk 3]
```diff
@@ -3,26 +3,27 @@
...
 
 
 def foo_three(a, g, j):
     """do foo three"""
     bat.do_a(a)
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

and didn't do a thing here that github can't see

Comment (zzzeek) [reply foo.py]: nope, I can see it, still in the diff

Comment (sqla-tester) [reply foo.py]: **mike bayer** (zzzeek) wrote:

touche

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

(1 comment)

ha

File: foo.py [hunk 4]
```diff
@@ -3,26 +3,27 @@
 from foo import bar
 from foo import bat
 
-import io  # noqa
 import re  # noqa
 
 
-def foo_one(x, y):
+def foo_one(x, y, z):
```
Comment (zzzeek) [inline foo.py]: review comment one

File: foo.py [hunk 5]
```diff
@@ -3,26 +3,27 @@
...
 
 def foo_two(z, x, y):
     """do foo two"""
     bat.do_z(z * 12)
     bar.do_x(x).y(y).z(z)
-    bar.do_y(y)
+    bar.do_y(y + 9)
 
 
 def foo_three(a, g, j):
```
Comment (zzzeek) [inline foo.py]: review comment two

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Code-Review+2 Workflow+1

you have done well

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1046 has been **merged**. Congratulations! :)

**Pull request closed without merging**